src/java/org/apache/cassandra/cql3/conditions/ColumnCondition.java

```diff
@@ -781,6 +781,10 @@ public ColumnCondition prepare(String keyspace, ColumnMetadata receiver, TableMe
             if (receiver.type instanceof CounterColumnType)
                 throw invalidRequest("Conditions on counters are not supported");
 
+            // Analyzer matches operator is only supported on SAI indexes for now
+            if (operator == Operator.ANALYZER_MATCHES)
+                throw invalidRequest(": operation can only be computed by an indexed column with a configured analyzer");
```

Comment:
I think that the error message is misleading. We do not support the use of indexes on LWT queries, which are the only one using Conditions so far. Therefore, even if you have an indexed column with a configured analyzer the query will not work. The message should be something like "Conditions do not support the : operator" .

Reply:
Could we be more specific and say "LWT Conditions"?  Many users probably don't realize the distinction between a Condition and a Predicate.